pkg/model/config/config_test.go

```diff
@@ -31,6 +31,43 @@ var _ = Describe("PluginConfig", func() {
 		Data2 string `json:"data-2"`
 	}
 
+	resource := Data{Group: "Foo", Kind: "Baz", Version: "v1"}
+	resource.Webhooks = Webhooks{false, false, true}
+
+	It("should return a new Webhooks", func() {
+		Expect(resource.Group).To(BeIdenticalTo("Foo"))
+		Expect(resource.Kind).To(BeIdenticalTo("Baz"))
+		Expect(resource.Version).To(BeIdenticalTo("v1"))
+		Expect(resource.Webhooks.Defaulting).To(BeIdenticalTo(false))
+		Expect(resource.Webhooks.Validation).To(BeIdenticalTo(false))
+		Expect(resource.Webhooks.Conversion).To(BeIdenticalTo(true))
+	})
+
+	It("should merge webhook when webhook is NOT equals", func() {
+		resourceOther := Data{Group: "Foo", Kind: "Baz", Version: "v1"}
+		resourceOther.Webhooks = Webhooks{false, false, false}
+		resource.merge(resourceOther)
+		Expect(resourceOther.Webhooks.Conversion).To(BeIdenticalTo(false))
+	})
+
+	It("should return true when has the Data is equals", func() {
+		Expect(resource.isGVKEqualTo(Data{Group: "Foo", Kind: "Baz", Version: "v1"})).To(BeIdenticalTo(true))
```

Comment:
NIT:  `BeTrue()` is clearer than `BeIdenticalTo(true)`
```suggestion
		Expect(resource.isGVKEqualTo(Data{Group: "Foo", Kind: "Baz", Version: "v1"})).To(BeTrue())
```